From be828543ac8f2815fb2863f1c18c1a25faef5fd7 Mon Sep 17 00:00:00 2001
From: Yehuda Yitschak <yehuday@marvell.com>
Date: Thu, 30 Jul 2015 10:58:17 +0300
Subject: [PATCH 0291/1240] fix: iob: added missing string in iob_dump

the configuration space window name was missing.
this caused a shift in all strings

Change-Id: I348484a389d566061ff3b8709c71ee569f29ba31
Signed-off-by: Yehuda Yitschak <yehuday@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/22329
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 drivers/misc/mvebu_iob.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/misc/mvebu_iob.c b/drivers/misc/mvebu_iob.c
index 8d29ced..8372305 100644
--- a/drivers/misc/mvebu_iob.c
+++ b/drivers/misc/mvebu_iob.c
@@ -128,8 +128,8 @@ void dump_iob(void)
 	u32 win_id, win_cr, alr, ahr;
 	u8 target_id;
 	u64 start, end;
-	char *iob_target_name[IOB_MAX_TID] = {"IHB0 ", "PEX1 ", "PEX2 ", "PEX0 ",
-						"NAND ", "RUNIT", "IHB1 "};
+	char *iob_target_name[IOB_MAX_TID] = {"CONFIG", "IHB0 ", "PEX1 ", "PEX2 ",
+					      "PEX0 ", "NAND ", "RUNIT", "IHB1 "};
 
 	/* Dump all IOB windows */
 	printf("bank  id target  start              end\n");
-- 
1.9.1

